features/step_definitions/find_car_steps.rb

```diff
@@ -0,0 +1,169 @@
+Given /^page with filters for search cars is opened$/ do
+  FindCarPage.open
+end
+
+When /^user fill in Postcode field with (\S+)$/ do |data|
+  FindCarPage.given.enter_data(data)
+end
+
+When /^user fill in Postcode field with correct data$/ do
+  FindCarPage.given.enter_data('E203EL')
+end
+
+And /^click the (.+) button$/ do |button_name|
```

Comment:
Don't use `And` word to describe steps, use `Given`, `When` or `Then`. This way it more clear of kind of step this Is